core/mondoo-ms365-security.mql.yaml

```diff
@@ -0,0 +1,625 @@
+policies:
+- uid: mondoo-ms365-security
+  name: Microsoft MS365 Security by Mondoo
+  version: 1.0.0
+  scoring_system: 2
+  authors:
+    - name: Mondoo, Inc
+      email: hello@mondoo.com
+  tags:
+    mondoo.com/platform: ms365,cloud
+    mondoo.com/category: security
+  docs:
+    desc: |-
+      ## Overview
+
+      MS365 Security by Mondoo provides guidance for establishing minimum recommended security and operational best practices for ms365.
+
+      ## Remote scan
+
+      Remote scans use native transports in `cnspec` to provide on demand scan results without the need to install any agents, or integration. 
+      
+      For a complete list of native transports run: 
+      
+      ```bash
+      cnspec scan --help
+      ```
+      ### Prerequisites
+
+      Remote scans of ms365 requires API credentials with access to the subscription. 
+
+      ### Scan an MS365 subscription
+
+      ```bash
+      cnspec scan ms365 --certificate-path <*.pem> --tenant-id <tenant_id> --client-id <client_id> --policy-bundle <*.mql.yaml>
+      ```
+
+      ## Join the community!
+
+      Our goal is to build policies that are simple to deploy, accurate, and actionable. 
+      
+      If you have any suggestions on how to improve this policy, or if you need support, [join the community](https://github.com/orgs/mondoohq/discussions) in GitHub Discussions. 
+  specs:
+  - asset_filter:
+      query: |
+        platform.name == "microsoft365"
+        platform.kind == "api"
+    scoring_queries:
+      mondoo-m365-ensure-multifactor-authentication-is-enabled-for-all-users-in-all-roles: null
```

Comment:
prefix should be `mondoo-ms365-security`

Reply:
Thanks Chris, done!